src/controllers/IllPersonController.ts

```diff
@@ -1,17 +1,89 @@
-import { controller, httpGet } from 'inversify-express-utils';
+import { SessionMiddleware } from 'ch-node-session-handler';
+import { controller } from 'inversify-express-utils';
+import { FormValidator } from './validators/FormValidator';
 
-import { BaseAsyncHttpController } from 'app/controllers/BaseAsyncHttpController';
+import { BaseController } from 'app/controllers/BaseController';
+import { AuthMiddleware } from 'app/middleware/AuthMiddleware';
 import { FeatureToggleMiddleware } from 'app/middleware/FeatureToggleMiddleware';
+import { loggerInstance } from 'app/middleware/Logger';
+import { Appeal } from 'app/models/Appeal';
+import { Illness } from 'app/models/Illness';
+import { OtherReason } from 'app/models/OtherReason.ts';
+import { IllPerson } from 'app/models/fields/IllPerson';
+import { schema } from 'app/models/fields/IllPerson.schema';
 import { Feature } from 'app/utils/Feature';
 import { ILL_PERSON_PAGE_URI } from 'app/utils/Paths';
+import { Navigation } from 'app/utils/navigation/navigation';
 
 const template = 'illness/ill-person';
 
-@controller(ILL_PERSON_PAGE_URI, FeatureToggleMiddleware(Feature.ILLNESS_REASON))
-export class IllPersonController extends BaseAsyncHttpController {
+const navigation : Navigation = {
+    previous(): string {
+        return '';
+    },
+    next(): string {
+        return '';
+    },
+    actions: (_: boolean) => {
+        return {
+            continue:'action=continue'
+        };
+    }
+};
+
+interface FormBody {
+    illPerson: IllPerson;
+    otherPerson: string;
+}
+
+@controller(ILL_PERSON_PAGE_URI, FeatureToggleMiddleware(Feature.ILLNESS_REASON),
+    SessionMiddleware, AuthMiddleware)
+export class IllPersonController extends BaseController<FormBody> {
+
+    constructor() {
+        super(
+            template,
+            navigation,
+            new FormValidator(schema)
+        );
+    }
+
+    protected prepareViewModelFromAppeal(appeal: Appeal): any {
+        const illness: Illness | undefined = appeal.reasons?.illness;
+        if (!illness) {
+            return {};
+        }
+
+        return {
+            illPerson: illness.illPerson,
+            otherPerson: illness.otherPerson
+        };
+    }
+
+    protected prepareSessionModelPriorSave(appeal: Appeal, value: any): Appeal {
+
+        const dummyOther: OtherReason = {
+            title: 'Dummy Reason',
+            description: 'The current Appeal data model requires an Other-type reason, ' +
+                'and reworking the data model is outside of the scope of this feature. ' +
+                'Until the Appeal object has been remodelled, this dummy reason must ' +
+                'be included.'
+        };
```

Comment:
I see why you did this, but I don't like it. Is there maybe another way of getting around it?

Reply:
`const emptyOther = {} as OtherReason`

would this pehaps work?

Reply:
I also don't like it. Unfortunately this is the only way to get around the issue until we rework the data structure, which is going to be a huge task. I cleared this with Damian on a previous ticket.

Reply:
did you try my suggestion?
`const emptyOther = {} as OtherReason`

Reply:
I originally did try setting `other: {}` but that didn't work. Turns out adding the `as OtherReason` to the end of it makes it work. TypeScript is _wild_. Thank you.